Documentation/user-guides/configuring-cluster-monitoring.md

```diff
@@ -1,9 +1,161 @@
 # Configuring Cluster Monitoring
 
-Parts of Cluster Monitoring are configurable. This configuration lies in a ConfigMap called `cluster-monitoring-config` in the `openshift-monitoring` namespace. The configuration file itself is defined under the `config.yaml` key within the ConfigMap's data.
+Parts of Cluster Monitoring are configurable. Depending which part of the stack the user wants to configure, he should edit the following: 
+
+- core infrastruture monitoring configuration lies in a ConfigMap called `cluster-monitoring-config` in the `openshift-monitoring` namespace.
+- user workload monitoring configuration lies in a ConfigMap called `user-workload-monitoring-config` in the `openshift-user-workload-monitoring` namespace.
+
+The configuration file itself is always defined under the `config.yaml` key within the ConfigMap's data.
 
 Configuring Cluster Monitoring is optional. If the config does not exist, or is empty or malformed, then defaults will be used.
 
+#### Note
+For simplicity in the following sections we will ommit the yaml necessary for the configuration to be persisted so for the example:
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    <FIELD_TO_BE_CONFIGURABLE>: <VALUE>
+```
+We will only show:
+```yaml
+<FIELD_TO_BE_CONFIGURABLE>: <VALUE>
+```
+
+## Core infrastruture monitoring
```

Comment:
nit: Core Infrastructure Monitoring

Reply:
the OCP docs say " Core OpenShift Container Platform monitoring components"